index.json

```diff
@@ -1 +1,410 @@
\ No newline at end of file
+{
```

Comment:
Can you please split this PR into two:

1. Readable JSON, no other changes.
2. Extra swagger you add to JSON and generate to Markdown